paragon/_fonts.scss

```
@@ -1 +1,4 @@
 // Add any font imports or definitions here
+@import url(https://fonts.googleapis.com/css?family=Open+Sans:600,400,300);
```

Google Fonts has an [improved API](https://developers.google.com/fonts/docs/css2) that we should probably be using. The specific URL is available from the [Open Sans specimen page](https://fonts.google.com/specimen/Open+Sans) on Google Fonts.

Also, should we include italic styles as well? It seems like it could be useful.

```suggestion
@import url('https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300;0,400;0,600;1,300;1,400;1,600&display=swap');
```

---

@michaelwheeler ah i didn't know that! Ok, i've updated the URL :)